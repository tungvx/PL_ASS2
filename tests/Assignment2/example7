ONE=one
TWO=ONE plus ONE
FOUR=TWO plus TWO
SIX=THREE plus THREE
THREE=three
%
FOUR equals four?
SIX equals six?