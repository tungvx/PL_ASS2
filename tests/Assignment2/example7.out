one plus one plus one plus one equals four?
THREE plus THREE equals six?